snap/info.go

```diff
@@ -78,6 +78,11 @@ func HookSecurityTag(snapName, hookName string) string {
 	return fmt.Sprintf("snap.%s.hook.%s", snapName, hookName)
 }
 
+// SecurityTag returns the snap-specific security tag.
+func SecurityTag(snapName string) string {
+	return fmt.Sprintf("snap.%s", snapName)
+}
```

Comment:
If we're going to have a function that does this, can we make sure we use it for the `AppSecurityTag()` and `HookSecurityTag()` functions so they can't diverge?